disco/state.py

```diff
@@ -153,6 +153,9 @@ def on_ready(self, event):
             self.dms[dm.id] = dm
             self.channels[dm.id] = dm
 
+    def on_user_update(self, event):
+        self.me = event.user
```

Comment:
This will generate stale references to the previous object. You want to do this insead:
```suggestion
       self.me.inplace_update(event.user)
```